gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/FolderCompletionContributor.java

```diff
@@ -31,19 +31,23 @@
 
 public class FolderCompletionContributor extends CompletionContributor {
 
-    public FolderCompletionContributor() {
-        extendCompletion(VersionPropsTypes.GROUP_PART);
+    private static final GradleCacheExplorer gradleCacheExplorer =
```

Comment:
this can actually just be an instance type, assuming you only have one instance of `FolderCompletionContributor`

Reply:
otherwise you've kinda shifted the problem around a bit. Ideally we only have mutable state as instance fields where possible. Sometimes this is unavoidable e.g. `ThreadLocal`s or where we don't control initialisation, but this doesn't seem to be the case here.

Reply:
yep should only be one instance